src/AppInstallerCommonCore/AppInstallerStrings.cpp

```diff
@@ -442,6 +444,13 @@ namespace AppInstaller::Utility
         return result;
     }
 
+    bool RegexFindAndReplace(std::string& inputStr, std::regex pattern, const char* value)

```

Comment:
For what you are using it (replace `\n` by `\n  `), I believe you can use `FindAndReplace()` above without resorting to using regex.